app/scripts/directives/mobileServiceSdkDocs.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileServiceSdkDocs', {
+    controller: [
+      "APIService",
+      "DataService",
+      MobileServiceSdkDocs
+    ],
+    controllerAs: 'ctrl',
+    bindings: {
+      context: '<',
+      mobileClient: '<',
+      serviceClasses: '<'
+    },
+    templateUrl: 'views/_mobile-service-sdk-docs.html'
+  });
+
+
+
+  function MobileServiceSdkDocs(APIService, DataService) {
+    var serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
+    var serviceInstanceVersion = APIService.getPreferredVersion('serviceinstances');
+
+    var ctrl = this;
+    var watches = [];
+
+    ctrl.$onChanges = function(changes) {
+      var clientChanges = _.get(changes, 'mobileClient.currentValue');
+      var serviceClassChanges = _.get(changes, 'serviceClasses.currentValue');
+
+      if (clientChanges && serviceClassChanges && watches.length === 0) {
+        watches.push(DataService.watch(serviceBindingsVersion, {namespace: ctrl.context}, function(bindingsData) {
+          ctrl.bindings = _.filter(bindingsData.by('metadata.name'), function(binding) {
+            return _.get(binding, ['metadata', 'annotations', 'binding.aerogear.org/consumer']) === _.get(ctrl, 'mobileClient.metadata.name');
```

Comment:
Same [annotations filter](https://github.com/openshift/origin-web-common/blob/master/src/filters/annotation.js#L4).

Reply:
You could potentially optimize by just `var name = _.get(ctrl, 'mobileClient.metadata.name')` before the `_.find()`, but I don't know that this is a huge deal.